docs/docs/runtime-api-reference.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: SDK
-sidebar_label: SDK
-slug: /SDK
+title: Runtime API Reference
```

Comment:
@galenmarchetti - NACK from me re: this name change. We should keep it as SDK.

Context: We previously had this section titled as `Client Libraries` and more than a few people asked me where our SDK was (Chainlink asked, and also a user asked too when we were doing a user feedback session). I changed the wording to SDK with Kevin's +1 [here](https://kurtosistech.slack.com/archives/C04MBCY49HR/p1686151655619479). Another data point we got recently was that our users search for the "Kurtosis SDK" on Google - https://github.com/kurtosis-tech/kurtosis/issues/1238

Reply:
Would prefer keeping the title as SDK as well

Reply:
+1

Reply:
@h4ck3rk3y @leeederek  Roger - I'm looking for a specifier that identifies why "SDK" is different than _starlark_. With just SDK we don't give any information on a first glance about why there's a difference between starlark api and SDK. Thoughts on "runtime SDK" or <some other word> SDK that clarifies why we have an SDK _and_ a starlark API?